### PR TITLE
Update quickstart for using v0.11

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
スクリーンショット 2023-06-28 17 27 00

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,6 +94,20 @@ <h3>Linux &amp; macOS</h3>
       <h3>Windows</h3>
       <pre><code>PowerShell -Command "&amp; {[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::TLS12; Invoke-WebRequest http://dl.embulk.org/embulk-latest.jar -OutFile embulk.bat}"</code></pre>
 
+      <p>In the Embulk v0.9, setup complete. Go to Next Steps.</p>
+
```
Comment (dmikurube) [inline index.html]: It'd be considerable to keep information for v0.9, but explaining both v0.9 and v0.11 could also confuse users.

I'm +1 to focus just on v0.11.

File: index.html [hunk 2]
```diff
@@ -94,6 +94,20 @@ <h3>Linux &amp; macOS</h3>
...
+      <h3>Install JRuby</h3>
+      <p>Download JRuby 9.x.y.z Complete jar from <a href="https://www.jruby.org">JRuby.org</a>. And put it in a favorite place..</p>
```
Comment (dmikurube) [inline index.html]:
```suggestion
      <h3>Set up with JRuby</h3>
      <p>Download your preferred version 9.x.y of JRuby Complete jar from <a href="https://www.jruby.org/download">JRuby Downloads</a>, and put it in your favorite directory.</p>
      <p>Embulk is tested with old JRuby 9.1.15.0 as of now.</p>
```

File: index.html [hunk 3]
```diff
@@ -94,6 +94,20 @@ <h3>Linux &amp; macOS</h3>
...
+      <h3>Setup embulk.properties</h3>
+      <p>Create the embulk.properties file in the $HOME/.embulk directory.</p>
+      <pre><code># embulk.properties
+jruby=file:///path/to/jruby-complete-9.x.y.z.jar</code></pre>
```
Comment (dmikurube) [inline index.html]:
```suggestion
      <p>Then, create `~/.embulk/embulk.properties` as follows.</p>
      <pre><code>jruby=file:///path/to/jruby-complete-9.x.y.z.jar</code></pre>
```

File: index.html [hunk 4]
```diff
@@ -94,6 +94,20 @@ <h3>Linux &amp; macOS</h3>
...
+      <h3>Install some gems</h3>
+      <pre><code>embulk gem install embulk
+embulk gem install msgpack
+embulk gem install bundler
+embulk gem install liquid</code></pre>
```
Comment (dmikurube) [inline index.html]:
```suggestion
      <p>Finally, install some Ruby gems.</p>
      <pre><code>embulk gem install embulk -v v0.11.0
embulk gem install msgpack
embulk gem install bundler  # if you need Bundler
embulk gem install liquid   # if you need Liquid</code></pre>
```

**Review by dmikurube — APPROVED** [review]
Thanks for working on it!

It basically looks good, but left some suggestions. Please let me know your thoughts on the suggestions.

File: index.html (hunk 1 above)
Comment (hiroyuki-sato) [reply index.html]: Thanks! Removed.

File: index.html (hunk 3 above)
Comment (hiroyuki-sato) [reply index.html]: Thanks! Fixed.

File: index.html (hunk 4 above)
Comment (hiroyuki-sato) [reply index.html]: Thanks!. Fixed.

**Comment by hiroyuki-sato** [comment]
@dmikurube 

Thanks! Fixed your comment.
What do you think to add the comment as follows?

```
embulk gem install embulk -v v0.11.0 # Must be the same Embulk version
```

**Comment by dmikurube** [comment]
> What do you think to add the comment as follows?

Sounds good! Let's go with that.

**Comment by hiroyuki-sato** [comment]
Thanks! I don't have merge permission. Please merge this.

**Pull request merged**